documentation/assemblies/upgrading/assembly-upgrade-cluster-operator.adoc

```diff
@@ -0,0 +1,48 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-upgrade.adoc
+
+[id='assembly-upgrade-cluster-operator-{context}']
+= Upgrading the Cluster Operator
+
+[role="_abstract"]
+How you upgrade the Cluster Operator depends on the way you deployed it.
```

Comment:
I don't think this translates well - it uses phrasal verbs and colloquialisms.

Please consider rewriting - maybe something like... "The procedure ( or replace with a better word, maybe steps) to upgrade the Cluster Operator depends on the initial methd of deployment." ?